mlrun/datastore/sources.py

```diff
@@ -948,11 +971,17 @@ def to_dataframe(self):
         if table_name and db_path:
             engine = db.create_engine(db_path)
             with engine.connect() as con:
-                return pd.read_sql(
-                    query,
-                    con=con,
-                    chunksize=self.attributes.get("chunksize"),
-                    parse_dates=self.attributes.get("time_fields"),
+                return filter_df_start_end_time(
+                    pd.read_sql(
+                        query,
+                        con=con,
+                        chunksize=self.attributes.get("chunksize"),
+                        parse_dates=self.attributes.get("time_fields"),
```

Comment:
Preexisting, but I'm pretty sure that `"time_fields"` (plural) is a bug, and it should be `"time_field"`.

Reply:
opened a ticket [ML-3929](https://jira.iguazeng.com/browse/ML-3929)